doc/nrf/app_dev/device_guides/nrf54l/cryptography.rst

```diff
@@ -0,0 +1,300 @@
+|                                      | ``PSA_KEY_PERSISTENCE_DEFAULT``,                                            |                                                                                                                               |
+|                                      | ``PSA_KEY_PERSISTENCE_READ_ONLY``,                                          |                                                                                                                               |
+|                                      | ``CRACEN_KEY_PERSISTENCE_REVOKABLE``,                                       |                                                                                                                               |
+|                                      | and location must be ``PSA_KEY_LOCATION_CRACEN_KMU``                        |                                                                                                                               |
++--------------------------------------+-----------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| key_id (psa_set_key_id)              | ``PSA_KEY_HANDLE_FROM_CRACEN_KMU_SLOT(kmu_usage_scheme, kmu_slot_nr)``      | For correct kmu_usage_scheme values, see :ref:`ug_nrf54l_crypto_kmu_key_usage_schemes`                                        |
+|                                      |                                                                             |                                                                                                                               |
+|                                      |                                                                             | For correct values for kmu_slot_nr, see :ref:`ug_nrf54l_crypto_kmu_slots`.                                                    |
++--------------------------------------+-----------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| key_usage (psa_set_key_usage_flags)  | Standard PSA Crypto key usage flags                                         | PSA_KEY_USAGE_EXPORT and PSA_KEY_USAGE_COPY are not allowed for keys with the usage scheme CRACEN_KEY_USAGE_SCHEME_PROTECTED. |
++--------------------------------------+-----------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------+
+
+.. _ug_nrf54l_crypto_kmu_key_usage_schemes:
+
+KMU key usage schemes
```

Comment:
Are there any of these users are not supposed to use? If so we should write that.
Is there one of these we recommend to use by default? I think we should recommend one so that users who really don't know end up picking something good and not just pick one at random

Reply:
Maybe we could even have a "typical use-case" column?

Reply:
If we list which schemes support which keys, and note that SEED is usually not relevant for users, this will be good.